tutorials/kafkaexactlyonce/src/main/resources/META-INF/properties.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<configuration>
+
+  <property>
+    <name>dt.application.KafkaExactlyOnceApp.operator.passthrough.prop.directoryPath</name>
+    <value>/kafka_exactly/</value>
```

Comment:
Suggest explicitly adding the path separator ('/') in the code rather than requiring it here.